server/sandbox_run.go

```diff
@@ -464,27 +464,28 @@ func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	var ip string
 	// setup the network
 	if !hostNetwork {
-		if err = s.netPlugin.SetUpPod(netNsPath, namespace, kubeName, id); err != nil {
-			return nil, fmt.Errorf("failed to create network for container %s in sandbox %s: %v", containerName, id, err)
+		podNetwork := newPodNetwork(sb.Namespace(), sb.KubeName(), sb.ID(), netNsPath)
+		if err = s.netPlugin.SetUpPod(podNetwork); err != nil {
+			return nil, fmt.Errorf("failed to create network for container %s in sandbox %s: %v", sb.Name(), id, err)
 		}
 
-		if ip, err = s.netPlugin.GetContainerNetworkStatus(netNsPath, namespace, id, kubeName); err != nil {
-			return nil, fmt.Errorf("failed to get network status for container %s in sandbox %s: %v", containerName, id, err)
+		if ip, err = s.netPlugin.GetPodNetworkStatus(podNetwork); err != nil {
+			return nil, fmt.Errorf("failed to get network status for container %s in sandbox %s: %v", sb.Name(), id, err)
```

Comment:
Maybe I'm misunderstanding this change, but should the error message be tweaked?  Perhaps:

"failed to get network status for sandbox %s with id %v".  Or are sb.Name and containerName equivalent?

If a change is indeed needed here, similar comment for the other error messages below.

Reply:
@TomSweeneyRedHat fixed